coalib/settings/ConfigurationGathering.py

```diff
@@ -240,9 +240,13 @@ def load_configuration(arg_list,
             Setting('config', re.escape(find_user_config(os.getcwd()))))
 
     targets = []
+    exclude_sections = []
     # We don't want to store targets argument back to file, thus remove it
     for item in list(cli_sections['cli'].contents.pop('targets', '')):
-        targets.append(item.lower())
+        if item[0] == '!':
+            exclude_sections.append(item[1:].lower())
```

Comment:
as you have hardcoded the `1` that originates from `len('!')`, I would recommend to put the `!` string into a constant `EXCLUDE_PREFIX` that you reuse at all relevant places to avoid maintenance problems.